src/app/pages/courses-page/courses-page.component.html

```diff
@@ -1 +1,7 @@
-<p>courses-page works!</p>
+<app-breadcrumbs></app-breadcrumbs>
+
+<section class="courses-page-container">
+  <app-actions-section></app-actions-section>
```

Comment:
it seems that app-actions-section might accept output handlers to inform parent container that something happened.

Reply:
Ok